docroot/legalcode/by-nc-nd_4.0.html

```diff
@@ -136,6 +136,7 @@
 <select>
 <option value="legalcode.ar">العربية</option>
```

Comment:
> > > > > > > - T,Me,blue,sol_AR**Do,and****Ass**_ SAM_itali**green**under,world**mica_mon><cia==coyote_luis%ch,t,m_ti_O**casa__dor**_**No,pued3%flay_dragon_avoidprivate::nom_Fonction*apache,nexus,linux,_,"list"tipo_Parametro__templates_pink**brown_A,R_1__**Root**Return**defecto**tipo,No**~~Alfrediño~~green```
> > El`A,CHA,S,M``~~GO`R,DO,O`~~`
````
```
```suggestion
 <option value="legalcode.ar">العربية</option>
```